.goreleaser.yml

```diff
@@ -13,9 +13,13 @@ builds:
     main: cmd/supernode/main.go
     goarch:
       - amd64
+      - arm64
     goos:
       - linux
       - darwin
+    ignore:
+      - goos: darwin
+        goarch: arm64
```

Comment:
Could you help me understand why this needs to be excluded?

Reply:
It is trying to get Darwin OS with ARM64 architecture, which is not available. So, I have added in ignore step to exclude Darwin with ARM64 build.